mp/bundle

```diff
@@ -2,19 +2,18 @@
 
 echo -e "\033[0;30;47m Bundling spec \033[0m"
 ${COMPOSE} run --rm bundler ash -c "
-mkdir -p /opt/dist
-json-refs resolve schema.json -f > /opt/spec/dist/swagger.json
+mkdir -p dist
+npm install -g json-refs
```

Comment:
I think it's better to keep this in the `Dockerfile`. Now we will have to install the `json-refs` package every time we want to bundle the spec.

Reply:
The bundler is only used when including the spec in another project. For development and to create the static site for the amazon bucket, we use `gulp-json-refs` in the node container. Do we want to create a custom Dockerfile which only runs 1 command which takes 2 seconds?

Reply:
I assumed npm would require a million packages and it would take more than 2 seconds to install json-refs. But I just tried it locally, and it does install pretty fast, so this seems ok.

Reply:
I do not have npm.

Reply:
`npm install -g json-refs` is being run in  the bundler container. But I see that @M4tini moved it to a Dockerfile again.